.github/CODEOWNERS

```diff
@@ -36,6 +36,10 @@
 
 /pkg/geo                     @cockroachdb/geospatial
 
+/pkg/kv                      @cockroachdb/kv
```

Comment:
and potentially unintended side effect of this is that teams will always be tagged if a piece of code touches their package. if it's not desired, we may have to append `@cockroachdb/kv+silent` or something so github doesn't do that.

Reply:
Yeah, that's a good idea. I think for KV we actually want that behavior, but as we backfill that file it's probably a good idea to do it in silent mode so that teams can opt into notifications rather than out (which requires a CI round-trip). I'll implement your suggestion.

Haha, doesn't work: @cockroachdb/kv+silent

Reply:
Or maybe it does in the file? It should. Anyway we'll figure out some scheme

Reply:
another alternative is to have our own ownership mechanism (e.g. the [OWNERS.yaml idea](https://github.com/cockroachdb/cockroach/pull/51880)), and have .github/CODEOWNERS be generated off that. You could have something like `review: false` or `review: true`, which could make github force a "request review" and put that file into CODEOWNERS

Reply:
I thought about that too, and maybe that's the way to go, but let's take the easy route for now. Code-gen'ning a CODOWNERS (which is order-sensitive) off a yaml sounds like it needs a lot of elbow grease to get right so I'd like to postpone it.

Reply:
Turns out any `+` symbol in any team name makes github ignore the whole codeowners file, so going to have to go with something else. I guess @teamname-noreview will have to do, going to do that tomorrow though. Should be ready for a look other than that!